packages/graph/hash_graph/bin/hash_graph/src/subcommands/mod.rs

```diff
@@ -0,0 +1,16 @@
+pub mod completions;
+pub mod migrate;
+pub mod server;
+
+use crate::subcommands::{completions::CompletionsArgs, migrate::MigrateArgs, server::ServerArgs};
```

Comment:
The `*Args` are not `pub`. Also, we could reexport the functions from the modules, but that's only a nit. Feel free to ignore!

Reply:
Yes, however, I'd prefer `self::` notatino here to disambiguate:
```rusts
pub use self::completions::CompletionsArgs;
pub use self::migrate::MigrateArgs;
pub use self::server::ServerArgs;
```
(rustfmt will do the rest for you)

Reply:
Sure, will let CI run and push a commit for this. Don't want to reset it again :D

Reply:
b2c1f2cf69ab22ba9752650deda363e22cc24a9f